File: Assignments/2/A3_NUTHANNAIK_CS19B051/ques2/tworay_parabolic_2.py
import math

def function():
	pt = 50.00
	pi = 2*math.asin(1)
	ht = 50.00
	hr = 2
	v = []
	lamba = (3.00)/9.00
	for dist in range(100, 1001, 200):
		gain = 10* math.log10(0.6*(((pi*3)/lamba)**2))
		pr = (pt *((hr*ht)**2)*gain)/(dist**4)
		pr = 30 + 10*(math.log10(pr))
		v.append(pr)
	freq2 = 100
	for i in range(len(v)):
		print(str(freq2) + " " + str(v[i]))
		freq2 += 200
	print()

if __name__ == '__main__':
	function()

File: Assignments/2/A3_AKHIL_CS19B053/3.2/tworay_parabolic_2.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
double Pt=50.00;
double Pi=2*asin(1);
double Ht=50;
double Hr=2;
double lamda=(double)(3/(double)9);
vector<double>v;
for(double dist=100;dist<=1000;dist+=200){
    double Pr;
     double Gain;
    Gain=10*log10((0.6*(pow(((Pi)*3)/lamda,2))));
    Pr=(double)(Pt)*(pow(Ht*Hr,2)*Gain)/pow(dist,4);
    Pr=30+10*log10(Pr);
    v.push_back(Pr);
}
int freq2=100;
int k=0;
for(int i=0;i<v.size();i++){
    cout<<freq2<<" "<<v[i]<<endl;
    freq2+=200;
}
cout<<endl;
}
